### PR TITLE
Ins 1711 blob storage in memory

### PR DESCRIPTION
**Pull request by aMFeriuSS**

### CODE REVIEW

File: ledger/storage/blob/blob.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	"context"
+
+	"github.com/insolar/insolar/core"
+)
+
+// Accessor provides info about Blob-values from storage
+//go:generate minimock -i github.com/insolar/insolar/ledger/storage/blob.Accessor -o ./ -s _mock.go
+type Accessor interface {
+	// Get returns Blob for provided id
+	Get(ctx context.Context, id core.RecordID) (Blob, error)
```
Comment (andreyromancev) [inline ledger/storage/blob/blob.go]: Let's call it "ForID", because there will be another one for jet.

File: ledger/storage/blob/blobmem.go [hunk 2]
```diff
@@ -0,0 +1,79 @@
+/*
```
Comment (andreyromancev) [inline ledger/storage/blob/blobmem.go]: This file doesn't have to begin with "blob". Let's call it just "memory" or "mem".

File: ledger/storage/blob/blob_cmp_test.go [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+			resBlob, err := blobStorage.Get(ctx, bl.id)
+			require.NoError(t, err)
+
+			assert.Equal(t, bl.b, resBlob)
+			assert.Equal(t, bl.b.Value, resBlob.Value)
+			assert.Equal(t, bl.b.JetID, resBlob.JetID)
+		}
+	})
+
+	t.Run("returns override error when saving with the same id", func(t *testing.T) {
```
Comment (andreyromancev) [inline ledger/storage/blob/blob_cmp_test.go]: This test fails if executed separately.

File: ledger/storage/blob/blob.go (hunk 1 above)
Comment (aMFeriuSS) [reply ledger/storage/blob/blob.go]: done

File: ledger/storage/blob/blobmem.go (hunk 2 above)
Comment (aMFeriuSS) [reply ledger/storage/blob/blobmem.go]: done

File: ledger/storage/blob/blob_cmp_test.go (hunk 3 above)
Comment (aMFeriuSS) [reply ledger/storage/blob/blob_cmp_test.go]: done

**Pull request merged**